### PR TITLE
updating output fields

### PR DESCRIPTION
**Pull request by gellwood**

### CODE REVIEW

File: v2_v3_outputs_map.csv [hunk 1]
```diff
@@ -178,15 +176,16 @@ Financial,replacement_costs,Financial,replacements_future_cost_after_tax
...
----,(Added in v3),Financial,year_one_om_costs_before_tax_bau
 Financial,year_one_om_costs_before_tax_bau_us_dollars,Financial,year_one_om_costs_before_tax_bau
-Financial,lcoe_component_diesel_capex_us_dollars_per_kwh,Financial,
-Financial,lcoe_component_fuel_us_dollars_per_kwh,Financial,
-Financial,lcoe_component_om_us_dollars_per_kwh,Financial,
-Financial,lcoe_component_other_annual_costs_us_dollars_per_kwh,Financial,
-Financial,lcoe_component_other_capex_us_dollars_per_kwh,Financial,
-Financial,lcoe_component_re_capex_us_dollars_per_kwh,Financial,
 Financial,total_om_costs_bau_us_dollars,Financial (not yet in v3),lifecycle_om_costs_after_tax_bau
+---,(Added in v3),Financial (not yet in v3),year_one_om_costs_before_tax_bau
```
Comment (adfarth) [inline v2_v3_outputs_map.csv:181]: @gellwood why is this repeated? (year_one_om_costs_before_tax_bau appears above as well)

Comment (gellwood) [reply v2_v3_outputs_map.csv:181]: @adfarth I think that's a mistake.

File: v2_v3_outputs_map.csv [hunk 2]
```diff
@@ -274,11 +271,6 @@ PV,pv_name,PV,name
 PV,size_kw,PV,size_kw
 PV,year_one_energy_produced_kwh,PV,year_one_energy_produced_kwh
 PV,year_one_energy_produced_bau_kwh,PV,year_one_energy_produced_kwh_bau
-PV,sr_provided_series_kw,PV (not in v3),
-PV,sr_required_series_kw,PV (not in v3),
-PV,station_distance_km,PV (not in v3),
-PV,station_latitude,PV (not in v3),
-PV,station_longitude,PV (not in v3),
 PV,year_one_power_production_series_kw,PV (not in v3),
```
Comment (adfarth) [inline v2_v3_outputs_map.csv:274]: @gellwood year_one_power_production_series_kw should show as "(Removed in v3)"

Comment (adfarth) [reply v2_v3_outputs_map.csv:274]: Also, one PV output is missing (a new output in v3): production_factor_series

**Comment by adfarth** [comment]
Just a few comments from me! Let me know when addressed and I will approve

File: v2_v3_outputs_map.csv (hunk 2 above)
Comment (gellwood) [reply v2_v3_outputs_map.csv]: @adfarth I just committed a new version of the outputs spreadsheet. Hopefully, I did it correctly. 

I addressed these comments as well as the duplicate from the above comment.

Here's the link https://github.com/NREL/REopt-Analysis-Scripts/pull/112/commits/8ce15b57900064f546228e254684d37bf4f0a98b

**Comment by adfarth** [comment]
Looks good! Approving!

**Review by adfarth — APPROVED** [review]

**Pull request merged**